src/test/java/systemtests/HelpCommandSystemTest.java

```diff
@@ -84,10 +84,15 @@ public void help_multipleCommands_onlyOneHelpWindowOpen() {
         executeCommand(HelpCommand.COMMAND_WORD);
 
         assertEquals(1, guiRobot.getNumberOfWindowsShown(HelpWindowHandle.HELP_WINDOW_TITLE));
-    }
+    }*/
 
     /**
-     * Asserts that the help window is open, and closes it after checking.
+     * Asserts that the help window is
+     *
+     *
+     *
+     *
+     * open, and closes it after checking.
```

Comment:
why so many spaces

Reply:
Spaces were created when running tests on intellij for some reason.